[2022-09-08T11:27:32.015+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: second_dag_v14.python_la_so_1 scheduled__2022-09-07T00:00:00+00:00 [queued]>
[2022-09-08T11:27:32.023+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: second_dag_v14.python_la_so_1 scheduled__2022-09-07T00:00:00+00:00 [queued]>
[2022-09-08T11:27:32.024+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-08T11:27:32.025+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 6
[2022-09-08T11:27:32.025+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-08T11:27:32.033+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): python_la_so_1> on 2022-09-07 00:00:00+00:00
[2022-09-08T11:27:32.037+0000] {standard_task_runner.py:52} INFO - Started process 1767 to run task
[2022-09-08T11:27:32.039+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'second_dag_v14', 'python_la_so_1', 'scheduled__2022-09-07T00:00:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmp7dj7n707', '--error-file', '/tmp/tmp2q79hnrl']
[2022-09-08T11:27:32.041+0000] {standard_task_runner.py:80} INFO - Job 36: Subtask python_la_so_1
[2022-09-08T11:27:32.079+0000] {task_command.py:371} INFO - Running <TaskInstance: second_dag_v14.python_la_so_1 scheduled__2022-09-07T00:00:00+00:00 [running]> on host e458dbb0fb9f
[2022-09-08T11:27:32.121+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=second_dag_v14
AIRFLOW_CTX_TASK_ID=python_la_so_1
AIRFLOW_CTX_EXECUTION_DATE=2022-09-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-07T00:00:00+00:00
[2022-09-08T11:27:32.123+0000] {logging_mixin.py:115} INFO - hello world
[2022-09-08T11:27:32.124+0000] {python.py:173} INFO - Done. Returned value was: None
[2022-09-08T11:27:32.134+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=second_dag_v14, task_id=python_la_so_1, execution_date=20220907T000000, start_date=20220908T112732, end_date=20220908T112732
[2022-09-08T11:27:32.375+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-08T11:27:32.403+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
